### ISSUE TITLE
Add selector-no-type

### ISSUE THREAD

**Issue by weaintplastic** [stylelint/stylelint#2252]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

I'd like to suggest adding a rule to disallow using types as selectors at all.

> Which rule, if any, is this issue related to?

This rule is kind of related to selector-no-qualifying-type but is even more strict.

> What CSS is should the rule be applied on?

```css
a {
  color: pink;
}
```

> What do you expect to happen?

Warn the user to not use types as selectors.

What do you think? Does it make sense for you to work on a rule like that or is there any other rule that could lint for the usage of types as selectors?

**Comment by weaintplastic** [comment]
Sorry. I figured out that the rule already exists.
